Applications/Hermes/hermesobject.py

```diff
@@ -0,0 +1,108 @@
+import hermesmessage
+import threading
+import Queue
+
+
+class TransmitObject(object):
+    """
+    This is the receiver object that provided receive function interface and buffering.
+
+    :param local_device_callsign: The callsign of the proxy assigned local Faraday device to connect to
+    :param local_device_node_id: The callsign ID number of the proxy assigned local Faraday device to connect to
+    """
+    def __init__(self, local_device_callsign, local_device_node_id):
+        self.local_device_callsign = local_device_callsign
+        self.local_device_node_id = local_device_node_id
+        # Create messaging application objects needed for transmissions
+        self.faraday_tx_msg_sm = hermesmessage.MsgStateMachineTx()
+        self.faraday_tx_msg_object = hermesmessage.MessageAppTx(self.local_device_callsign,
+                                                                self.local_device_node_id)
+        # Create receiver application object
+        self.faraday_rx_msg_object = hermesmessage.MessageAppRx()
+        self.rx_uart_service_port_application_number = 3
```

Comment:
These are things you really don't want to mess with unless you really know what you're doing and go change a lot of other code/firmware.

I created global CONSTANTS:

![image](https://cloud.githubusercontent.com/assets/339144/24601497/fc51a0bc-180d-11e7-849d-fc879b839b7c.png)